### PR TITLE
Remove reference to `create_index()` for `VectorStoreDriver`s

### LINKED ISSUE #207: Remove references to `create_index()` in `VectorStoreDriver` docs

### PR DESCRIPTION
**Pull request by vachillo**
- [x] create variables for integ tests
closes #207 

----
📚 Documentation preview 📚: https://griptape--205.org.readthedocs.build//205/

### CODE REVIEW

**Review by vachillo — COMMENTED** [review]
there is already a note about the number of dimensions needed when creating an index. is that sufficient? should we link to the corresponding docs pages for creating indexes?

**Comment by collindutter** [comment]
Maybe we should just add a note to that indicating that they should check with their respective DB's index creation process.

That said, a couple do require some non-intuitive setup. I've got these written down maybe we could include:

Redis: 
```
FT.CREATE idx:griptape ON hash PREFIX 1 "griptape:" SCHEMA tag TAG vector VECTOR FLAT 6 TYPE FLOAT32 DIM 1536 DISTANCE_METRIC COSINE
```
Mongo:
```
{
  "fields": [
    {
      "numDimensions": 1536,
      "path": "vector",
      "similarity": "euclidean",
      "type": "vector"
    },
    {
      "path": "namespace",
      "type": "filter"
    }
  ]
}
```

File: docs/examples/multiple-agent-shared-memory.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ MONGODB_USERNAME = os.getenv("MONGODB_USERNAME")
 MONGODB_PASSWORD = os.getenv("MONGODB_PASSWORD")
 MONGODB_DATABASE_NAME = os.getenv("MONGODB_DATABASE_NAME")
 MONGODB_COLLECTION_NAME = os.getenv("MONGODB_COLLECTION_NAME")
+MONGODB_INDEX_NAME = os.getenv("MONGODB_INDEX_NAME")
+MONGODB_VECTOR_PATH = os.getenv("MONGODB_VECTOR_PATH")
```
Comment (collindutter) [inline docs/examples/multiple-agent-shared-memory.md:28]: I think we need to add these to the integration test env file right?

File: .github/workflows/integration-tests.yml [hunk 2]
```diff
@@ -38,6 +38,15 @@ jobs:
...
       MONGODB_PASSWORD: ${{ secrets.INTEG_MONGODB_PASSWORD }}
+      MONGODB_INDEX_NAME: ${{ secrets.MONGODB_INDEX_NAME }}
+      MONGODB_VECTOR_PATH: ${{ secrets.MONGODB_VECTOR_PATH }}
+      AZURE_MONGODB_HOST: ${{ secrets.AZURE_MONGODB_HOST }}
+      AZURE_MONGODB_USERNAME: ${{ secrets.AZURE_MONGODB_USERNAME }}
+      AZURE_MONGODB_PASSWORD: ${{ secrets.AZURE_MONGODB_PASSWORD }}
+      AZURE_MONGODB_DATABASE_NAME: ${{ secrets.AZURE_MONGODB_DATABASE_NAME }}
+      AZURE_MONGODB_COLLECTION_NAME: ${{ secrets.AZURE_MONGODB_COLLECTION_NAME }}
+      AZURE_MONGODB_INDEX_NAME: ${{ secrets.AZURE_MONGODB_INDEX_NAME }}
+      AZURE_MONGODB_VECTOR_PATH: ${{ secrets.AZURE_MONGODB_VECTOR_PATH }}
```
Comment (collindutter) [inline .github/workflows/integration-tests.yml:49]: Secrets are prefixed with `INTEG_`

Comment (vachillo) [reply .github/workflows/integration-tests.yml:49]: oops

**Review by collindutter — DISMISSED** [review]

**Review by collindutter — APPROVED** [review]

**Pull request merged**